### DISCUSSION TITLE
gesp-knowledge-scanf-printf-floatnumber/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#351] [General]
# gesp-knowledge-scanf-printf-floatnumber/

最近做基础练习时频繁使用到scanf和printf函数。在对double类型变量进行输入输出时发现，scanf标准为 %lf 而printf标准为 %f ，并且printf实际使用 %lf 和 %f ，效果似乎一样，遂想弄清一二。

https://www.coderli.com/gesp-knowledge-scanf-printf-floatnumber/

**Comment by Niuger** [comment]
2024.9.29打卡
